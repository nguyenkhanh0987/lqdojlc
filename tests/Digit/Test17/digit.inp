100
77942226 462614210
653032294 663584946
177679914 585243127
606264074 743530576
251249957 904259291
749069320 796350911
628685048 695822837
465880436 699903298
168659986 383217068
127892224 182993752
847411723 873726303
144587656 215771884
87996103 998235285
495352086 942955447
951482814 988468462
93064803 199747546
235668233 685065887
413687676 582162213
437581660 933774991
774018566 830469857
278270726 592156261
862374773 864488793
529043544 747477627
701925267 937713024
46934234 309099975
176717380 602703072
918885680 930772799
249811153 962395666
603185058 950153301
941663801 956965086
378756668 637530791
856846341 962146926
253208597 460216612
122739012 289075224
530270513 996723838
575157904 855612720
617600568 678379136
903298475 988261593
603640318 761875726
359175414 498604560
102876678 255499893
166307948 720440322
781660200 997829419
452501016 517507082
238390857 717706808
877748930 971498827
302937365 799172159
585926977 940247749
669055638 770003360
301334023 654337682
104154328 490692643
83224411 172751748
35998677 353993240
88960644 260733703
184300450 618828919
247484754 306949219
810452779 864402618
896639339 930529882
691174352 845131234
990374577 999489715
714951636 735624739
112094008 571595105
634704684 841291156
37658415 879614300
251577712 714518602
96236474 876968730
677380304 722617649
189251956 541186591
672031568 971511007
844058957 998717706
634093416 933050445
640488645 726880562
901014029 931530560
960180516 995800183
595787589 915907692
780643329 821799053
120543080 393813674
671437103 809122840
350626567 363951890
954352036 971301943
959073986 960558342
589854137 651879443
5282410 494517223
84205196 244404148
62414108 253921577
177401177 286636908
468046970 525426924
417094968 763546320
686459988 930749699
550269367 672345699
852464309 907637251
637372520 681243872
248580099 366039783
29742824 893651107
281518494 665802207
393542112 825789871
498579097 987940178
275082561 840088985
920187871 948699252
627914271 674339434
