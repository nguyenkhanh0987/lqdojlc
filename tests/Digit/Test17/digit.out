64527093
1174369
61365593
12017834
77970378
5854463
8272036
28566165
37015389
11233610
2252989
14482710
112792297
40099812
3025319
21019180
61893258
21166331
49387788
4450057
44776534
232606
22673820
16812651
50549324
64396526
344644
80596406
27425927
859940
32389191
7065448
31705897
30916420
41947217
25773072
5825620
4580680
13639179
21984380
26161283
77531650
15816324
9627960
64624957
5810441
59358800
29699731
9306041
44470976
62908256
16882351
58162129
30549475
63136294
11997506
3011276
970360
11991351
1138565
1019623
70875555
18413945
116112910
62402779
101722038
4538761
52183558
23391177
11428587
25097255
9066499
728376
3575326
27095928
3552633
48292032
13447403
2457869
1110124
71291
5437830
84772697
29067592
36334905
19929718
7872608
38343329
18828493
13926671
5163535
4959387
19536105
119701535
50518022
46543165
42887996
67025330
1111172
4776206
